README.md

```diff
@@ -1,74 +1,105 @@
-# Experimental development repository of Subsocial and it's pallets
+# SubSocial Full Node by [DappForce](https://github.com/dappforce)
 
-## Build
+SubSocial is a set of Substrate runtime libraries (called FRAME) with UI that would allow anyone to launch their own decentralized censorship-resistant social network aka community. We are planning to follow a topology of Polkadot Network where every community will be running on its own Substrate chain and all these decentralized communities will be connected to our own Polkadot relay. This social networking relay could be connected to the official Polkadot Network.
 
-Install Rust:
+You can think of this as decentralized versions of Reddit, Stack Exchange or Medium, where subreddits or communities of Stack Exchange or blogs on Medium run on their own chain. At the same time, users of these decentralized communities should be able to transfer or share their reputation, coins and other values from one community to another via Polkadot relay.
 
+## Building from source
+
+### Initial setup
+If you want to build from source you will need the Rust [toolchain](https://rustup.rs/), openssl and llvm/libclang:
 ```bash
 curl https://sh.rustup.rs -sSf | sh
+sudo apt install make clang pkg-config libssl-dev
+rustup update
+```
+Then, you should have cloned Subsocial Substrate repository:
+```bash
+git clone https://github.com/dappforce/dappforce-subsocial-node
 ```
 
-Initialize your Wasm Build environment:
-
+### Building
+Initialise the WASM build environment:
 ```bash
+cd dappforce-subsocial-node/
 ./scripts/init.sh
 ```
 
-Build Wasm and native code:
-
+Build the node (native code):
 ```bash
 cargo build --release
 ```
 
-## Run
-
-### Single Node Development Chain
+### Running a public node
+[!] Experimental and may not work. Better run [with Docker](#connect-to-our-network-with-docker)
+Run the node and connect to the public Subsocial Testnet.
+```bash
+./target/release/subsocial-node --name "Node Name" --pruning archive --bootnodes /ip4/167.172.104.62/tcp/30333/p2p/QmfHoYER3gPchRvHSTzyFZ9hhvQG3oP9jgcQEbfbm2x9LR /ip4/167.172.104.62/tcp/30334/p2p/QmVcp8pBKtKcwWmWcrARZcdLxW9E1EzfU9JwvgiSKu4How
+```
 
-Purge any existing developer chain state:
+### Installing a release build
+This will install the executable `subsocial-node` to your `~/.cargo/bin` folder, which you would normally have in your `$PATH` environment.
```

Comment:
```suggestion
This will install an executable `subsocial-node` to your `~/.cargo/bin` folder, which you would normally have in your `$PATH` environment.
```